benchmark/torch/qmix/train.py

```diff
@@ -0,0 +1,188 @@
+#   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from smac.env import StarCraft2Env
+from env_wrapper import SC2EnvWrapper
+from replay_buffer import EpisodeExperience, EpisodeReplayBuffer
+from qmixer_model import QMixerModel
+from rnn_model import RNNModel
+from qmix import QMIX
+from qmix_agent import QMixAgent
+import parl
+from parl.utils import logger
+from parl.utils import summary
+import numpy as np
+
+logger.set_dir('./log_path')
+
+config = {
```

Comment:
Maybe we can put the config to the file `qmix_config.py`

Reply:
modified.